acct-enricher-job/src/test/scala/com/ubirch/testers/JobTester.scala

```diff
@@ -0,0 +1,79 @@
+package com.ubirch.testers
+
+import com.typesafe.config.ConfigFactory
+import com.ubirch.models.postgres.{EventDAO, FlywaySupport, IdentityDAO, JobDAO, TenantDAO}
+import com.ubirch.services.{AcctEventsService, DailyCountResult}
+import com.ubirch.services.externals.{Identity, Tenant, ThingAPI}
+import com.ubirch.{InjectorHelper, InjectorHelperImpl, Job, TestBaseWithH2}
+import monix.eval.Task
+import org.mockito.ArgumentMatchers
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+
+import java.time.LocalDate
+import java.util.UUID
+import scala.concurrent.duration.DurationInt
+
+class JobTester extends TestBaseWithH2 with MockitoSugar {
```

Comment:
This class should not be in this package as this package is for actual little testers, but not for tests. Please move it to the proper place and rename the class to something like JobTest or JobSpec